platform/error/platform-error-impl/src/main/resources/codeMessages.properties

```diff
@@ -7,12 +7,10 @@
 # See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
 # <http://www.gnu.org/licenses/lgpl.html>.
 
-400=Bad request. If you are not already signed in, please \
-  <a href="../login/index.html?prevurl={{uri}}">Sign In</a> and try again.
-401=You are not authenticated. If you are not already signed in, please \
-  <a href="../login/index.html?prevurl={{uri}}">Sign In</a> to continue.
-403=You are not authorized to see the resource you are attempting to view. If you are not already \
-  signed in, please <a href="../login/index.html?prevurl={{uri}}">Sign In</a> and try again.
+400=Bad request.
+401=You are not authenticated.
+403=You are not authorized to see the resource you are attempting to view. If you are logged in \
```

Comment:
> If you are logged in to a different user

✏️ I wouldn't expect a user to have multiple accounts with different permissions, so telling the user "logout and try again" will only help if they're browsing as guest. Wondering if it makes sense to change this to "if you logged in as guest..."

Reply:
The guest thing should be transparent to users imo. We shouldn't advertise that "guest" even exists. We could say "if you have not logged in, please logout and try again." Or something along those lines.

Reply:
It is kind of weird to say if you have not logged in, log out